packages/api/src/hooks/useAccountSecurity.ts

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import useAuthorize from './useAuthorize';
+import useRequest from '../useRequest';
+
+type TAccountSecurityPayload = NonNullable<
+    NonNullable<Parameters<ReturnType<typeof useRequest<'account_security'>>['mutate']>[0]>['payload']
+>;
+
+const useAccountSecurity = () => {
+    const { isSuccess: success_auth } = useAuthorize();
+    const { mutate, ...rest } = useRequest('account_security');
```

Comment:
`useRequest` is deprecated, and now `useMutation`.
https://deriv-group.slack.com/archives/C3AHMSM3P/p1695697505953689?thread_ts=1670994836.610179&cid=C3AHMSM3P

Reply:
will need a pull from master for the feature branch for it